data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+mission "Hai Tour [2] Alta Hai"
+	landing
+	name "Visit the ringworld on <planet>"
+	description "Go to <destination> and learn more about the Hai's worlds."
+	source "Stonebreak"
+	destination "Alta Hai"
+	to offer
+		has "ZZ Hai Tour [1] Stonebreak: done"
+	on offer
+		conversation
+			`This star system has an unusual number of metallic asteroids, which you noticed the Hai harvesting for the raw materials before you landed. Stonebreak itself is home to incredibly large cities: massive, clearly planned communities built in concentric rings, where housing space and community buildings alternate with zones of factories and shipyards.`
+			`	From above, this city has almost perfect radial symmetry, a sure sign that it was planned and built as a single unit rather than expanding at random over time. Every single building in the city is a dull and slightly irregular gray color, all made from the same material which seems to be some metallic alloy, and the corners of each building are slightly beveled instead of meeting at right angles.`
+			`	Even the spaceport is symmetrical: a ring of hangars face outwards at evenly spaced intervals, and warehouses and living quarters dot the inside of the ring.`
+			`	While the layout is utilitarian, it looks to be an incredibly comfortable place to live; the Hai have clearly given their individual community spaces quite a lot of character and life.`
+			`	As you are wandering around and getting a feel for the place, you overhear someone talking about an apparently complete ringworld in Alta Hai. It definitely sounds like something you should check out.`
+				accept
+
+	#Goto Stop 3 Wah Yoot (Darkcloak) - must happen after Quarg first contact
+mission "ZZ Hai Tour [3] Meet Unfettered"
+	landing
+	name "Meet the Rebels on <planet>"
+	description "Go to <destination> and learn more about the Hai's conflict."
+	source "Alta Hai"
+	destination "Darkcloak"
+	to offer
+		has "Hai Tour [2] Alta Hai: done"
+	on offer
+		conversation
+			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships while housing storage facilities for cargo.`
+			`	Pondering your questions, you find a Quarg and ask, "So this isn't Hai space then, but it's a dead end?"`
+			`	The Quarg explains, "Much like in Enif in human space, this is where we live as friends to the Hai."`
+			`	This explanation seems typically Quarg, but it's the best one you're able to get. Before you go, you ask, "Do you know about the rebels I've been hearing about?"`
+			`	The Quarg responds, "The Hai and their rebels are the same. You should travel to Darkcloak in Wah Yoot to understand."`
+			`	It seems like there's nothing else for it but to see for yourself.`
```

Comment:
This tour doesn't feel very organic and seems unnecessary to me. I'd say it could probably be removed and any information that it's suppose to provide to the player should be provided in later missions where necessary. If nothing else I'd say it needs heavily reworked to actually give something of value to the player outside of just saying "Oh check out this cool thing, not check out that cool thing," and you should use alternate versions of the first contacts where necessary instead of fudging the mission names to fiddle with the offer orders and having multiple missions stack on top of each other.

Reply:
The value it provides to the player (who: reminder, only gets this if they have never met the Hai) is that they fill out the map of all the Hai systems as prelude to needing to know where most of them are later on for plot reasons.

And also, this gives them a solid chance to simply look around and experience the Hai space before being thrust into the narrative.

Reply:
Such goals can still be achieved without feeling so inorganic. As I said, "If nothing else I'd say it needs heavily reworked..."